app/src/marketplace/containers/ProjectEditing/ProjectController.test.tsx

```diff
@@ -108,35 +106,23 @@ const PROJECT_STUB: Project = {
 
 describe('ProjectController', () => {
     let controller: ProjectController
-    let persistOperations: PersistOperation[] = []
 
     const prepareTestForProjectCreate = (
         createProjectResult: boolean,
         validationResult: Partial<Record<RecursiveKeyOf<Project>, {level: validationCtx.SeverityLevel, message: string}>>,
         state: Project,
     ) => {
-        persistOperations = [];
         (createProject as Mock).mockReset();
         (useProjectState as Mock).mockReset();
-        (createProject as Mock).mockImplementation(() => ({
-            onTransactionComplete: ((cb: () => void) => {
-                if (createProjectResult === true) {
-                    cb()
-                }
-            }),
-            onError: ((cb: () => void) => {
-                if (createProjectResult === false) {
-                    cb()
-                }
-            })
-        } as any))
+        (createProject as Mock).mockImplementation(() => new Promise<void>((resolve, reject) => {
+            createProjectResult ? resolve() : reject()
+        }))
```

Comment:
We no longer use `send` (from `smartContract.ts`) here. In general, there's more power in not using it.